### DISCUSSION TITLE
development/2020/07/19/opensource-analysis/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#127] [Comments]
#  Python 오픈소스(Open Source) 분석 방법

https://zzsza.github.io/development/2020/07/19/opensource-analysis/

**Comment by zzsza** [comment]
<p>학부때를 마지막으로, 바쁘다는 핑계로 맨날 미뤄왔는데 좋은 자극 받고 갑니다.<br>시간나면 하는 것이 아니라, 틈틈히 시간을 만드려고 노력해야하는 것 같아요</p>

>_This comment was made by Disqus. 2021-05-05 05:19:18 키모띠앙_

**Comment by zzsza** [comment]
<p>현직 DE입니다.<br>좋은 글 감사합니다 ㅎㅎ</p>

>_This comment was made by Disqus. 2022-02-16 10:28:34 DeadLine_

**Comment by jaepil-choi** [comment]
혼자하다 3번 산으로 가고 좀 좋은 오픈소스 베스트 프랙티스를 배워야겠다 했는데 탁 펴보니 막막하더라고요

좋은 기운 얻고 갑니다.

**Comment by cestbonciel** [comment]
오픈소스 갖다 쓰기만 했었어요.  코드를 열어 살펴보고 싶긴 한데, 어떻게 해야하나 막막했는데 저의 가이드라인이 되어줄 것 같아요! 감사합니당

**Reply by cestbonciel** [reply]
저도 오픈소스 열어서 파악해보고 블로그에도 한번 기재해봐야겠어요!
